app/core/src/main/java/com/fsck/k9/K9.kt

```diff
@@ -171,7 +171,7 @@ object K9 : EarlyInit {
     var isChangeContactNameColor = false
 
     @JvmStatic
-    var contactNameColor = 0xff00008f.toInt()
+    var contactNameColor = -638932 // #F6402C
```

Comment:
Why use the decimal notation? The advantage of the hexadecimal notation is that you can easily read the value of the alpha, red, green, and blue components. It feels especially odd, when you add a comment for the hexadecimal color notation anyway.

Side note:
In general we try to avoid comments that either duplicate information or that explain how something is done. Comments tend not to be updated when the code is changed and then there are conflicting statements.

Sometimes it's helpful to know *why* something was done. In that case a comment with the rationale can be appropriate.